model/vulnerability/registry.yaml

```diff
@@ -0,0 +1,120 @@
+groups:
+  - id: registry.vulnerability
+    type: attribute_group
+    display_name: Vulnerability
+    brief: >
+      The vulnerability fields describe information about a vulnerability.
+    attributes:
+      - id: vulnerability.category
+        type: string[]
+        stability: experimental
+        brief: >
+          The type of system or architecture that the vulnerability affects. These may be platform-specific
+          (for example, Debian or SUSE) or general (for example, Database or Firewall). For example
+          [Qualys vulnerability categories](https://qualysguard.qualys.com/qwebhelp/fo_portal/knowledgebase/vulnerability_categories.htm).
+        examples:
+          - ['Firewall', 'Database']
+      - id: vulnerability.classification
+        type: string
+        stability: experimental
+        brief: >
+          The classification of the vulnerability scoring system. For example, [CVSS](https://www.first.org/cvss/specification-document).
+        examples: ['CVSS']
+      - id: vulnerability.description
+        type: string
+        stability: experimental
+        brief: >
+          The description of the vulnerability that provides additional context of the vulnerability.
+          For example [Common Vulnerabilities and Exposure CVE description](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['A vulnerability in the firewall allows an attacker to bypass security controls.']
+      - id: vulnerability.enumeration
+        type: string
+        stability: experimental
+        brief: >
+          The type of identifier used for this vulnerability. For example, [Common Vulnerabilities and Exposure CVE enumeration](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['CVE', 'SNYK']
+      - id: vulnerability.id
+        type: string
+        stability: experimental
+        brief: >
+          A unique identification number for the vulnerability.
+          For example, [Common Vulnerabilities and Exposure CVE ID](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['CVE-2021-44228', 'SNYK-JS-LODASH-567123']
+      - id: vulnerability.reference
+        type: string
```

Comment:
Should this be `string[]` (it is described as a "list of resources")?

Reply:
hey @kelnage I have updated the description, it should be the main link to the vulnerability as defined in the vulnerability classification